frontend/public/components/utils/kebab.tsx

```diff
@@ -103,34 +103,69 @@ export const ResourceKebab = connectToModel((props: ResourceKebabProps) => {
   />;
 });
 
-export class Kebab extends DropdownMixin {
+export class Kebab extends React.Component<KebabProps, KebabState> {
   static factory: KebabFactory = kebabFactory;
-  private onClick = this.onClick_.bind(this);
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false
+    };
+  }
 
-  onClick_(event, option) {
-    event.preventDefault();
+  onToggle = isOpen => {
+    this.setState({
+      isOpen
+    });
+  };
 
-    if (option.callback) {
-      option.callback();
-    }
+  onSelect = event => {
+    this.setState({
+      isOpen: !this.state.isOpen
+    });
+  };
 
-    if (option.href) {
-      history.push(option.href);
+  render() {
+    const { isOpen } = this.state;
+    const { options, isDisabled, position, id } = this.props;
```

Comment:
Are you referring to the "Dropdown panel" example here?
http://patternfly-react.surge.sh/patternfly-4/components/dropdown/

This renders the children as panel contents instead of passing the `dropdownItems` prop:
```
      <Dropdown
        onSelect={this.onSelect}
        toggle={<DropdownToggle onToggle={this.onToggle}>Expanded Dropdown</DropdownToggle>}
        isOpen={isOpen}
      >
        <div>[Panel contents here]</div>
      </Dropdown>
```

Just confirming w/ you first...

Reply:
I mean this kebap. So the user doesn't have to create its own controlled component with the right toggle. And so it is possible to pass custom dropdown Items

Reply:
added a patch for comments above https://github.com/priley86/web-ui/commit/ef61ec1d184e2fa5ea71bb05b2cce8ad773b0fd8 , let me know if this makes sense to you and I'll apply it...

Reply:
+1, looks good